components/invest/BedIndex.tsx

```diff
@@ -1,111 +1,122 @@
-                from='#FF1013'
-                to='#000000
-'>
-                The only index that can get ETH and BTC maxis in BED together
-              </Gradient>
+              )}
+            </Container>
+            <br />
+            <Text as={'span'} color={'red.400'}>
+              The only index that can get ETH and BTC maxis in BED together
             </Text>
-          </Container>
-          <Container maxW='container.xl'>
+          </Heading>
+          <Stack spacing={{ base: 4, sm: 6 }}>
             <Text fontWeight={400} lineHeight='22px' fontSize='lg'>
-              The only index that can get ETH and BTC maxis in BED together The
-              Bankless BED Index is a passive investment vehicle weighted
+              The Bankless BED Index is a passive investment vehicle weighted
               equally in Bitcoin, Ethereum, and the DeFi Pulse Index (DPI),
               which tracks high-usage DeFi assets. This index gives you exposure
               to crypto without having to research DeFi projects or constantly
               rebalance your portfolio. This is a great first crypto investment
               for anyone just starting their crypto journey. The fund is a
               cooperation between BanklessDAO and Index Coop, a provider of DeFi
-              investment products
+              investment products.
             </Text>
-          </Container>
-          <Container
-            display='flex'
-            justifyContent='space-evenly'
-            alignItems={'center'}
-            flexDirection={{ base: 'column', md: 'row' }}>
-            <Button
-              margin='10px 0px;'
-              onClick={() =>
-                window.open('https://app.indexcoop.com/bed', '_blank')
-              }>
-              <Container padding={0} paddingRight={2}>
-                <Image
-                  src='/images/bed-token.png'
-                  alt='bed-token'
-                  height={23}
-                  width={23}
-                />
-              </Container>
-              Invest in BED index
-            </Button>
-            <Button
-              onClick={() =>
-                window.open(
-                  'https://www.bankless.community/bed-index',
-                  '_blank'
-                )
-              }>
-              Learn more about Bed Index{' '}
-            </Button>
-          </Container>
-        </VStack>
-        {!isMobile ? (
-          <VStack h='full' w='full' p={10} spacing={10} align='flex-end'>
-            <Image
-              src='/images/bed-token.png'
-              alt='bed-token'
-              height={454}
-              width={454}
-            />
+          </Stack>
+          <VStack justifyContent={'flex-end'} display='flex'>
+            <Container
+              display='flex'
+              justifyContent='space-evenly'
+              alignItems={'center'}
+              flexDirection={{ base: 'column', md: 'row' }}>
+              <Button
```

Comment:
MINOR : most of the button here seem to be links - we should probably have them as links to make the markup semantic

Reply:
Also they have that odd onClick => window.open code we've seen elsewhere, we should replace with Link